chart/templates/_helpers.yaml

```
@@ -664,6 +664,17 @@ server_tls_key_file = /etc/pgbouncer/server.key
   {{- include "_serviceAccountName" (merge (dict "key" "triggerer") .) -}}
 {{- end }}
 
+{{/* Set the triggerer capacity to use */}}
+{{- define "triggerer.capacity" -}}
+  {{- $capacity := .Values.config.triggerer.capacity | default 0 | int -}}
+  {{- $defaultCapacity := .Values.config.triggerer.default_capacity | default 1000 -}}
+  {{- if gt $capacity 0 -}}
+    {{- $capacity -}}
+  {{- else -}}
+    {{- $defaultCapacity -}}
+  {{- end -}}
```

```suggestion
  {{- $triggerer_section := .Values.config.triggerer | default dict }}
  {{- $triggerer_section.capacity | default ($triggerer_section.default_capacity | default 1000) | int -}}
```

When we remove the default from values.yaml, this will keep the helper working.